<launch>
  <arg name="robot_name"   default="open_manipulator_6dof"/>
  <arg name="end_effector" default="gripper"/>

  <group ns="$(arg robot_name)">
    <node name="motion" pkg="open_manipulator_motion" type="open_manipulator_motion" output="screen">
      <remap from="kinematics_pose" to="$(arg end_effector)/kinematics_pose"/>
    </node>
  </group>
</launch>
